server/src/main/java/com/lunchtime/models/Feedback.java

```diff
@@ -0,0 +1,62 @@
+package com.lunchtime.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Setter;
+import lombok.Getter;
+import org.hibernate.annotations.ColumnDefault;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+
+@Entity
+@Setter
+@Getter
+public class Feedback {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    @Column(name = "description", columnDefinition = "TEXT")
+    private String description;
```

Comment:
Also PostgreSQL [provides](https://www.postgresql.org/docs/9.3/datatype-character.html) text type, it _stores strings of any unlimited length_. Due to feedback origin, it can't be "long story". So, we can use simple `@Column(length=140)`, for example

Reply:
description can be more big 140 symbol.TEXT it's better

Reply:
Check this **[link](https://www.postgresql.org/docs/9.3/datatype-character.html)** again, please. `TEXT` type isused for storing very big texts. There is no necessity of its usage due to our simple usage `@Column(length=140)` or `@Column(length=1140)`, it doesn't matter.

Moreover, `TEXT` type in not SQL standart and other databases that don't provide it may conflict if switch them. Yes, you'll remind me YAGNI, but, in fact, we don't need `TEXT` type here. 

And, the **[most](https://www.postgresql.org/docs/9.3/datatype-character.html)** important: _If you desire to store long strings with no specific upper limit, use text or character varying without a length specifier, rather than making up an arbitrary length limit.)_ According to my survey, `TEXT` type doesn't have any restrigtion for its length.

Reply:
![image](https://user-images.githubusercontent.com/50665431/77145381-0979c980-6a91-11ea-9c05-ba033aec47c5.png)